algo/src/main/java/com/hartwig/serve/sources/ckb/CkbExtractorFactory.java

```diff
@@ -14,10 +15,20 @@ private CkbExtractorFactory() {
     }
 
     @NotNull
-    public static CkbExtractor buildCkbExtractor(@NotNull EventClassifierConfig config, @NotNull RefGenomeResource refGenomeResource,
+    public static CkbExtractor createEvidenceExtractor(@NotNull EventClassifierConfig config, @NotNull RefGenomeResource refGenomeResource,
             @NotNull TreatmentApproachCurator treatmentApproachCurator) {
         // We want to capture all events from CKB, so ignore driver inconsistencies
-        return new CkbExtractor(EventExtractorFactory.create(config, refGenomeResource, DriverInconsistencyMode.IGNORE),
-                treatmentApproachCurator);
+        return new CkbExtractor(Knowledgebase.CKB_EVIDENCE,
+                EventExtractorFactory.create(config, refGenomeResource, DriverInconsistencyMode.IGNORE),
```

Comment:
I suggest to create the event extractor in a separate function that is reused between both evidence and trial extractor creation (along with the comment on why we set the driver inconsistency mode to `IGNORE`). This way you are guaranteed that event extraction is always identical between trials and evidence

Reply:
changed this